src/options.html

```diff
@@ -9,29 +9,70 @@
 </head>
 
 <body id="options-body">
-  <div id="container">
+  <div id="container" class="grid-with-banner">
+    <!-- Main Tab Content + Donation plea -->
     <header>
-
-      <!-- Navigation Tabs -->
-      <nav>
-        <div class="nav nav-tabs" id="nav-recap-tab" role="tablist">
-          <a id="nav-recap-tab" class="col-sm text-center nav-item nav-link active" data-toggle="tab" role="tab"
-            href="#nav-recap" aria-controls="nav-recap" aria-selected="true">
-            RECAP Archive
-          </a>
-          <a id="nav-settings-tab" class="col-sm text-center nav-item nav-link" data-toggle="tab" role="tab"
-            href="#nav-settings" aria-controls="nav-settings" aria-selected="false">
-            Settings
-          </a>
-          <a id="nav-donate-tab" class="col-sm text-center nav-item nav-link" data-toggle="tab" role="tab"
-            href="#nav-donate" aria-controls="nav-donate" aria-selected="false">
-            Donate
-          </a>
+      <div class="d-flex flex-column">
+        <div id="header-banner" class="info-banner d-flex flex-column">
+          <div id="message-section" class="d-flex">
+            <div style="padding-top:12.5px; padding-right:20px; padding-bottom:12.5px" class="flex align-items-center">
```

Comment:
Why put style code into the HTML instead of the CSS file?

Reply:
Sorry, I should've removed the "style" attribute while adding the new CSS classes to the banner. I often experiment with the style tag before moving things to classes. I'm cleaning those up now